scripts/merge-and-build.sh

```diff
@@ -175,10 +175,9 @@ if [ "${OSE_VERSION}" != "3.2" ] ; then
   VC_COMMIT="$(GIT_REF=enterprise-${OSE_VERSION} hack/vendor-console.sh 2>/dev/null | grep "Vendoring origin-web-console" | awk '{print $4}')"
   git add pkg/assets/bindata.go
   git add pkg/assets/java/bindata.go
-  set +e # Temporarily turn off errexit. THis is failing sometimes. Check with Troy if it is expected.
-  # This fails only when there is nothing new to commit, which is normal.
-  git commit -m "[DROP] bump origin-web-console ${VC_COMMIT}"
-  set -e
+  if [ "$(git status --porcelain)" ]; then
```

Comment:
> If this is in the rebase script, though, again, won't we always have changes since we're blowing away old changes and OCP never tracks Origin Web Console master?

I am stashing the previous web console diff so up to the point we revendor the console, the directory should be clean. Also, [OCP master is tracking Origin Web Console master](https://github.com/openshift/aos-cd-jobs/blob/fa928f42996b1566f2f06e39cba1f277509ab402/scripts/merge-and-build.sh#L108).